### PR TITLE
Add test for invoke headers

### PR DESCRIPTION
**Pull request by LucasRoesler**
**What**
- Remove CI flow for Swarm because the project i snow archived
- Add check for the X-Call-Id header when invoking a function
- Add check for the X-Start-Time header when ivoking a function

Covers the coming changes to the gateway https://github.com/openfaas/faas/pull/1627

Signed-off-by: Lucas Roesler <roesler.lucas@gmail.com>

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -28,32 +28,6 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.29
-  test-swarm:
```
Comment (alexellis) [inline .github/workflows/test.yaml:31]: Thanks for removing this.

**Comment by alexellis** [comment]
The chart is up to date now, so I've kicked off the job for K8s again.

**Comment by alexellis** [comment]
LGTM

The next question would be - which tests won't pass with faasd and do you have any suggestions on how to add it to the certifier?

**Comment by LucasRoesler** [comment]
> LGTM
> 
> The next question would be - which tests won't pass with faasd and do you have any suggestions on how to add it to the certifier?

RE: adding faasd, we just need a contrib script for setup and install during the CI. I assume we will just have to clone it and use `./hack/install.sh`  per https://github.com/openfaas/faasd#deploy-faasd

**Comment by alexellis** [comment]
I would say so. Can it be done in a separate "job" in parallel? docker's containerd + stand-alone containerd don't play well together. What is the best way to do this with GitHub Actions?

**Pull request merged**